site/gatsby-site/react-table-config.d.ts

```diff
@@ -0,0 +1,120 @@
+import {
```

Comment:
Is this file needed? I'm curious about the file path `site/gatsby-site`. It looks inappropriate

Reply:
Yes, it's to avoid [this](https://stackoverflow.com/questions/60006656/react-table-not-assignable-to-type-partialtablestateobject) issue, following the solution provided by the package [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-table/Readme.md).

Reply:
OK, I got it! I wonder if we can watch that issue in the `react-table` for a possible official fix.

Reply:
I agree with the weird location comment, I think the convention is to have them inside a `/types` folder.